/simple_controller:
  ros__parameters:

    error_margin: 0.003

    #friction

    # Fv_gain_0: 0.
    # FFa_gain_0: 0.
    # Kd_gain_0: 0.1
    # Ki_gain_0: 10.
    # Kp_gain_0: 80.0
    # Coulomb_friction_0: 0.
    # i_clamp_h_0: 10.
    # i_clamp_l_0: -10.


    FFv_gain_0: 0.0
    FFa_gain_0: 0.
    Kd_gain_0: 50.0
    Ki_gain_0: 320.0
    Kp_gain_0: 640.0
    Coulomb_friction_0: 0.
    i_clamp_h_0: 10.
    i_clamp_l_0: -10.

    FFv_gain_1: 0.0
    FFa_gain_1: 0.
    Kd_gain_1: 50.0
    Ki_gain_1: 320.0
    Kp_gain_1: 640.0
    Coulomb_friction_1: 0.
    i_clamp_h_1: 10.
    i_clamp_l_1: -10.

    FFv_gain_2: 0.0
    FFa_gain_2: 0.
    Kd_gain_2: 50.0
    Ki_gain_2: 320.0
    Kp_gain_2: 640.0
    Coulomb_friction_2: 0.
    i_clamp_h_2: 10.
    i_clamp_l_2: -10.

    FFv_gain_3: 0.0
    FFa_gain_3: 0.
    Kd_gain_3: 50.0
    Ki_gain_3: 320.0
    Kp_gain_3: 640.0
    Coulomb_friction_3: 0.
    i_clamp_h_3: 10.
    i_clamp_l_3: -10.

    FFv_gain_4: 0.0
    FFa_gain_4: 0.
    Kd_gain_4: 50.0
    Ki_gain_4: 320.0
    Kp_gain_4: 640.0
    Coulomb_friction_4: 0.
    i_clamp_h_4: 10.
    i_clamp_l_4: -10.

    FFv_gain_5: 0.0
    FFa_gain_5: 0.
    Kd_gain_5: 50.0
    Ki_gain_5: 320.0
    Kp_gain_5: 640.0
    Coulomb_friction_5: 0.
    i_clamp_h_5: 10.
    i_clamp_l_5: -10.

    use_sim_time: false

    # FFv_gain_0: 12.0
    # Kd_gain_0: 20.0
    # Ki_gain_0: 1.0
    # Kp_gain_0: 30.0
    # Coulomb_friction_0: 12.5